src/krylov/pcg.c

```diff
@@ -98,7 +98,10 @@ hypre_PCGCreate( hypre_PCGFunctions *pcg_functions )
    (pcg_data -> rel_change)   = 0;
    (pcg_data -> recompute_residual) = 0;
    (pcg_data -> recompute_residual_p) = 0;
+   (pcg_data -> rtol)         = 0.0;
```

Comment:
This line and the next are already set above (lines 95 and 97)

Reply:
BTW, I'm not sure what `rtol` actually does.  It looks like it is only used in the `ams_driver.c` file.  I wonder if anybody really uses this.  It checks the difference in successive residuals, so it's similar to rel-change in the sense that it is looking for the residual norm to flatten out.